src/server/game/Server/WorldSession.cpp

```diff
@@ -255,7 +249,6 @@ void WorldSession::SendPacket(WorldPacket const* packet)
         return;
     }
 
-    LOG_TRACE("network.opcode", "S->C: {} {}", GetPlayerInfo(), GetOpcodeNameForLogging(static_cast<OpcodeServer>(packet->GetOpcode())));
```

Comment:
I don't think it's worth deleting it

Reply:
The packet is already logged to file one call later, in [WorldSocket::SendPacket](https://github.com/azerothcore/azerothcore-wotlk/blob/062591c3c4a5978e5c18c7a4dfb2ae6ea6a5c321/src/server/game/Server/WorldSocket.cpp#L428).

Reply:
Good to have it sent in console too.

Reply:
> Good to have it sent in console too.

What's the point in showing it in the console when the screen is getting spammed with tens of lines per second under normal debugging condition? It's not even a relevant message to begin with...